### PR TITLE
Update manuals.yml (YML)

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Copy paste is bad

## Why / Balance
Clean yml

## How to test
N/A

## Media
N/A

## Breaking changes
N/A

**Changelog**
N/A

### CODE REVIEW

**Review by whatston3 — COMMENTED** [review]
Scanned through changes, looks good, will quickly test.

**Comment by dvir001** [comment]
Merge conflicts go brrrrr

File: Resources/Prototypes/_NF/Catalog/Fills/Paper/Shipyard/manuals.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 - type: entity
   parent: BaseItem
-  id: ShipyardBookwormInfo
-  name: bookworm user manual
+  id: ShipyardInfoBase
+  name: shuttle user manual
   description: preflight checklist
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/Fills/Paper/Shipyard/manuals.yml:5]:
```suggestion
  description: The preflight checklist for a shuttle. Check it for power settings!
```

File: Resources/Prototypes/_NF/Catalog/Fills/Paper/Shipyard/manuals.yml [hunk 2]
```diff
@@ -17,50 +18,32 @@
...
+  - type: GuideHelp
     guides:
     - ShipyardBookworm
 
 - type: entity
-  parent: BaseItem
+  parent: ShipyardInfoBase
   id: ShipyardBrigandInfo
   name: brigand user manual
   description: preflight checklist
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/Fills/Paper/Shipyard/manuals.yml:35]:
```suggestion
```

**Review by whatston3 — CHANGES_REQUESTED** [review]
I'd request that the names of the ships be capitalized in the name if you care.

**Review by whatston3 — APPROVED** [review]
Thank you.

**Pull request merged**